packages/amplify-cli-core/src/feature-flags/featureFlags.ts

```diff
@@ -0,0 +1,31 @@
+export type FeatureFlagsEntry = { [key: string]: any };
```

Comment:
nit could use `Record<string, any>`
Also, since we're always storing flags with a "section" and a "feature" and since the value can only be a string, number or boolean wouldn't a better type be
```
Record<string, Record<string, number | boolean | string>>
```

Reply:
good catch